scss/components/calendar.scss

```diff
@@ -152,6 +142,9 @@ $fd-calendar-table-width: $fd-calendar-table-item-width * 7 !default;
     align-items: center;
   }
   &__action {
+    & > * {
+      @include fd-reset-child-spacing();
+    }
```

Comment:
I added this mixin to use in action-type containers that will keep its children from blowing out with crazy margins.  I want to reuse this in `action-bar` as well.